lib/fog/compute/google/requests/get_image_from_family.rb

```diff
@@ -0,0 +1,35 @@
+module Fog
+  module Compute
+    class Google
+      # Returns the latest non-deprecated image that is part of an image family.
+      #
+      # ==== Parameters
+      # * family<~String> - Name of the image resource to return.
+      #
+      # ==== Returns
+      # * response<~Excon::Response>:
+      #   * body<~Hash> - corresponding compute#image resource
+      #
+      # ==== See also:
+      # https://cloud.google.com/compute/docs/reference/latest/images/getFromFamily
+      class Mock
+        def get_image_from_family(_family, _project = @project)
+          Fog::Mock.not_implemented
```

Comment:
@plribeiro3000 Yeah, we still have it in many places. Should I remove it from everywhere then?